cli/lib/replace.ts

```diff
@@ -0,0 +1,19 @@
+import through2 from 'through2'
+import type { Quad } from '@rdfjs/types'
+import $rdf from 'rdf-ext'
+
+function replace(version: string) {
+  const prefix = 'https://cube.link/latest/'
+  const replacement = `https://cube.link/${version}/`
+  return through2.obj(function (quad: Quad, _, done) {
+    if (quad.predicate.value === 'https://code.described.at/imports' && quad.object.value.startsWith(prefix)) {
```

Comment:
```suggestion
    if (quad.predicate.equals(env.ns.code.imports && quad.object.value.startsWith(prefix)) {
```

Of course you'd need to get `env` from step context